service/provider/src/main/java/gov/va/vro/service/provider/mas/service/MasProcessingService.java

```diff
@@ -62,8 +70,14 @@ public String processIncomingClaim(MasAutomatedClaimPayload payload) {
       offRampClaim(payload, offRampReason);
       return offRampReason;
     }
-    camelEntrance.notifyAutomatedClaim(
-        payload, masConfig.getMasProcessingInitialDelay(), masConfig.getMasRetryCount());
+
+    var headers =
+        Map.of(
+            MasIntegrationRoutes.MAS_DELAY_PARAM,
+            masConfig.getMasProcessingInitialDelay(),
+            MasIntegrationRoutes.MAS_RETRY_PARAM,
+            masConfig.getMasRetryCount());
+    camelEntry.inOnly(IMVP_EXCHANGE, NOTIFY_AUTOMATED_CLAIM_QUEUE, payload, headers);
```

Comment:
Inject payload into the `NOTIFY_AUTOMATED_CLAIM_QUEUE` (RabbitMQ) because the `camelEntrance.notifyAutomatedClaim()` is repeated used by MasPollingProcessor for the same payload. The route for the `NOTIFY_AUTOMATED_CLAIM_QUEUE` saves the payload to a file, which only needs to be done once. This RabbitMQ queue also exposes the route outside the container/JVM so that we can resubmit/inject payloads.